### PR TITLE
134 - dig-a-hash.com added!

### PR DESCRIPTION
**Pull request by KidSysco**
## Description

Adding Dig-A-Hash to the organisations.yml file by request of @Eomm from Twitts...

https://twitter.com/ManuEomm/status/1688056049457135616

## Related Issues

This one looked right, but not entirely sure.

https://github.com/fastify/website/issues/135

## Check List

- [X] I have read the [Contributing Guidelines](./CONTRIBUTING.md)

### CODE REVIEW

File: static/img/organisations/dig-a-hash.png
Comment (Eomm) [inline static/img/organisations/dig-a-hash.png:1]: Could you upload an svg version?
I don't think we support png at the moment :doh:

**Comment by KidSysco** [comment]
No, we do not have a version of that logo in SVG format.

File: static/img/organisations/dig-a-hash.png
Comment (KidSysco) [reply static/img/organisations/dig-a-hash.png:1]: No, we do not have a version of that logo in SVG format.

**Comment by KidSysco** [comment]
I did what you asked to the best of my ability. If that is not good enough, then you can just close this PR. Thanks.

**Review by Eomm — APPROVED** [review]
it works 
image

**Pull request merged**